common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -93,17 +93,48 @@
     <%doc>
         Loads a studio-frontend page, with the necessary context. Context is expected
         as a dictionary in the body of this tag.
-
-        Dev note: we could also add the locale-injection script in this block
-            -use a better default than the hardcoded 'en'. There should be a setting or something?
-            -lookup (webpack exported) locale-injection script using lang as key
-            -include it as the first script in this block
     </%doc>
     <%
         body = capture(caller.body)
+        body_dict = json.loads(body)
+        locale = body_dict['lang']
+        messages = "{}"
+        localeData = ""
+
+        if locale != 'en':
```

Comment:
We could export `en` explicitly if we need to; it would make this code a bit simpler. I don't really have a preference